src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -518,6 +521,8 @@ def load_arguments(self, _):
                    help='set ephemeral disk storage pool option for azure container storage')
         c.argument('node_provisioning_mode', is_preview=True, arg_type=get_enum_type(node_provisioning_modes),
                    help='Set the node provisioning mode of the cluster. Valid values are "Auto" and "Manual". For more information on "Auto" mode see aka.ms/aks/nap.')
+        # in creation scenario, use "localuser" as default
+        c.argument('ssh_access', type=str, default=CONST_SSH_ACCESS_LOCALUSER, is_preview=True, validator=validate_ssh_access)
```

Comment:
could omit `type=str` as this is the default type, same for 3 other uses

Reply:
updated